pkg/executables/cmk.go

```diff
@@ -200,7 +318,12 @@ func (c *Cmk) exec(ctx context.Context, args ...string) (stdout bytes.Buffer, er
 	if err != nil {
 		return bytes.Buffer{}, fmt.Errorf("failed cmk validations: %v", err)
 	}
-	argsWithConfigFile := append([]string{"-c", configFile}, args...)
+	argsWithConfigFile := []string{"-c", configFile}
+	for _, arg := range args {
```

Comment:
what's the point of filtering out the spaces?

Reply:
removed. 

The reason it was added there is for unit testing which compares command args with expected values.